### ISSUE TITLE
Files do not use encryption

### ISSUE THREAD

**Issue by exil0867** [movim/movim#1331]
I have this issue where text messages do get encrypted. But files do not.
This issue currently occurs on https://mov.im instance (v0.25.1).

**Comment by exil0867** [comment]
@edhelas Do you any idea about this?

**Comment by edhelas** [comment]
Yes, it's not planned for now as I have other plans at the moment.
If you're willing to implement it, I'm always happy to have a nice PR to review :)

**Comment by Strxct** [comment]
Heyy I am new to movim but been a developer for a while I saw that you had an encryption for images in the movim/src/movim/image.php 

couldnt you use this globally for all the extension and use an if statement to sort out the files from the images making it easier to implement this?

and I would like to know this so I could generate a PR with a potential fix

I hope to hear from you soon :)

**Comment by WillFrancois** [comment]
I'm interested in providing some form of support for OMEMO file encryption following "XEP-0454: OMEMO Media sharing". Is this still something that would benefit the project? Is this an appropriate extension or would another XEP be more suitable for this case?
